vba_timeseries/models/exp/m_suuvid_obs_exp.m

```diff
@@ -0,0 +1,48 @@
+function  [ gx ] = m_suuvid_obs_exp(Xt, phi, u, inG)
+% INPUT
+% - x_t : hidden states (weights of basis functions)
+% - phi : temperature (1x1)
+% - u   : imput vector (not used in observation)
+% - inG : multinomial
+% OUTPUT
+% - gx : p(chosen|x_t)
+
+phi = transform_phi(phi, inG);
+gamma = phi(1); %slope on logistic (sensitivity)
```

Comment:
Should probably update the comment to be 'value-based recovery component'

Reply:
I was pointing to gamma as the value-based recovery component. Basically it scales how much motor speed/probability recovers as a function of the value available (Q*)

Reply:
I see. I just changed that comment.